packages/manager/apps/hpc-vmware-managed-vcd/src/components/tiles/organization-options-tile/OptionsTileLicense.tsx

```diff
@@ -0,0 +1,34 @@
+import { ActionMenu, Description } from '@ovhcloud/manager-components';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { useParams } from 'react-router-dom';
+import useManagedVcdOrganization from '@/data/hooks/useManagedVcdOrganization';
+import TileSubtitle from '../tile-subtitle/TileSubtitle.component';
+
+export default function OptionsTileLicense() {
+  const { t } = useTranslation('dashboard');
+  const { id } = useParams();
+  const { data: vcdOrganization } = useManagedVcdOrganization(id);
```

Comment:
Loading and error state management are missing

Reply:
no fix needed here ?

Reply:
Data is now passed using props instead this hook, but I forgot to delete this component (not used anymore), thanks 👍